Framework/CurveFitting/src/Functions/CrystalFieldMultiSpectrum.cpp

```diff
@@ -122,15 +122,13 @@ void CrystalFieldMultiSpectrum::init() {
 }
 
 size_t CrystalFieldMultiSpectrum::getNumberDomains() const {
-  if (!m_target) {
+  if (!m_target)
```

Comment:
Is removing the braces a requirement of CppCheck? David and I had a discussion about these kinds of `if` statements as they can make it easier to introduce bugs. There's quite an interesting post about it where using these caused a security flaw in an old version of OSX, the "goto fail" bug: https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/

Reply:
No its not a requirement of cppcheck, I'll add them back in. Thats an interesting read, thanks!